apps/dashboard/src/components/activity/constants.ts

```diff
@@ -0,0 +1,92 @@
+import { JobStatusEnum, StepTypeEnum } from '@novu/shared';
+import { RiForbidFill } from 'react-icons/ri';
+import { RiErrorWarningLine } from 'react-icons/ri';
+import { BadgeVariant } from '../primitives/badge';
+import { RiCheckboxCircleLine, RiLoader3Line, RiLoader4Fill } from 'react-icons/ri';
+import { IconType } from 'react-icons/lib';
+
+export const STEP_TYPE_LABELS: Record<StepTypeEnum, string> = {
+  [StepTypeEnum.EMAIL]: 'Email',
```

Comment:
I am sure this constant exists elsewhere! I've seen it 10 times :p

Reply:
I couldn't find anything related :) We have one for icons, maybe that's the one you've seen

Reply:
Please then move to the new reusable constant.